iam.tf

```diff
@@ -52,3 +52,19 @@ EOF
 
   tags = "${var.apiary_tags}"
 }
+
+resource "aws_iam_role_policy" "secretsmanager_for_ranger_task" {
+  name = "secretsmanager"
```

Comment:
`secretsmanager ` --> `ranger_secretsmanager ` ?

Reply:
This is already qualified for IAM role name, this is just inline policy, so current config should be fine.